README.md

```diff
@@ -28,29 +31,80 @@ once.
+`AirshipAdapter.shared(context).restore()` in `Application.onCreate()` so that the Gimbal SDK can
+process Gimbal Place Events reliably when the app is restarted from terminated state.
+
 ### Android Marshmallow Permissions
 
 Before the adapter is able to be started on Android M, it must request the location permission
-``ACCESS_FINE_LOCATION``. The adapter has convenience methods that you can use to request permissions while
-starting the adapter:
-```
+`ACCESS_FINE_LOCATION`. The adapter has convenience methods that you can use to request permissions
+while  starting the adapter:
+
+```java
     AirshipAdapter.shared(context).startWithPermissionPrompt("## PLACE YOUR GIMBAL API KEY HERE ##");
 ```
 
 Alternatively you can follow [requesting runtime permissions](https://developer.android.com/training/permissions/requesting.html)
-to manually request the proper permissions. Then once the permissions are granted, call start on the adapter.
+to manually request the proper permissions. Then once the permissions are granted, call start on
+the adapter.
 
 Note: You will need `ACCESS_BACKGROUND_LOCATION` permissions to use Gimbal's background features
 
+## Enabling Event Tracking
+By default, event tracking is disabled, and thus must be explicitly enabled as described below.
+
+### RegionEvents
+To enable or disable the tracking of Airship RegionEvent objects, use the shouldTrackRegionEvents property:
+
+AirshipAdapter.shared.shouldTrackRegionEvents = true // enabled
+AirshipAdapter.shared.shouldTrackRegionEvents = false // disabled
+
+### CustomEvents
+To enable or disable the tracking of Airship CustomEvent objects, use the shouldTrackCustomEntryEvents and shouldTrackCustomExitEvents properties to track events upon place entry and exit, as shown below. For more information regarding Airship Custom Events, see the documentation here.
+
+```java
+    // To enable CustomEvent tracking for place exits
+    AirshipAdapter.shared(UAirship.getApplicationContext()).setShouldTrackCustomExitEvent(true);
+
+    // To disable CustomEvent tracking for place exits
+    AirshipAdapter.shared(UAirship.getApplicationContext()).setShouldTrackCustomExitEvent(false);
+    
+    // To enable CustomEvent tracking for place entries
+    AirshipAdapter.shared(UAirship.getApplicationContext()).setShouldTrackCustomEntryEvent(true);
+    
+    // To disable CustomEvent tracking for place entries
+    AirshipAdapter.shared(UAirship.getApplicationContext()).setShouldTrackCustomEntryEvent(false);
+```
+
 ## Stopping the adapter
 
 Adapter can be stopped at anytime by calling:
-```
+
+```java
    AirshipAdapter.shared(context).stop();
 ```
+
+Once `stop()` is called, Gimbal location event processing will not restart upon subsequent app
+starts.
+
+## AirshipGimbalAdapter Migration
+
+update gradle dependency to `com.gimbal.android.v4:airship-adapter:1.0.0`
+update all references to the `AirshipGimbalAdapter` class should be changed to `AirshipAdapter`
```

Comment:
If you're trying to mirror the iOS adapter README, I think the only missing bit is the final, final bullet point explaining to customers that if they wanna keep the behavior exactly the same after migrating, then they're going to want to turn region event-tracking ON, and custom event-tracking OFF. Otherwise, looks pretty good to me.